File: codility.com/C/lesson1-1.c
int solution(int A[], int N) {
    // write your code in C99
    int sum = 0, cur = 0, min = -1;
    int i = 0;
    
    for (i = 0; i < N; i++){
       sum += A[i];   
    }
    
    for(i = 0; i < N-1; i++){
       cur += A[i];
       if(abs(sum - 2 * cur) < min || min == -1){
          min = abs(sum - 2 * cur);
       }
    }
    return min;
}

File: codility.com/Python/lesson1-1.py
# you can write to stdout for debugging purposes, e.g.
# print "this is a debug message"

def solution(A):
    # write your code in Python 2.7
    summa = sum(A)
    cur = 0
    minimum = None
        
    for elem in A[:-1]:
        cur += elem
        if minimum is None or minimum > abs(summa - 2 * cur):
            minimum = abs(summa - 2 * cur)
        
    return minimum
